splunk_connect_for_snmp_poller/manager/data/Inventory.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from dataclasses import dataclass
+
+
+@dataclass
+class Inventory:
```

Comment:
Actually inventory is a collection of agents configuration. Maybe we should rename it to AgentConfig or InventoryEntry or sth similar.